packages/scratch-gui/webpack.config.js

```diff
@@ -105,6 +105,12 @@ const buildConfig = baseConfig.clone()
         },
         output: {
             path: path.resolve(__dirname, 'build')
+        },
+        resolve: {
+            alias: {
+                'react': require.resolve(path.join(__dirname, '../../node_modules/react')),
+                'react-dom': require.resolve(path.join(__dirname, '../../node_modules/react-dom'))
+            }
```

Comment:
I have forgotten to remove this code. It was for the local setup to be able to link scratch-paint to gui.